get_directory_sizes.py

```diff
@@ -59,11 +63,12 @@ def main():
 
 
 def get_folder_sizes(
-    artifactory_url, repositories,
+    artifactory_url, repositories, user=None, password=None,
     verbose=False, num_workers=10
 ):
     url = '%s/api/application.wadl' % (artifactory_url,)
-    resp = requests.head(url, timeout=5)
+    auth = HTTPBasicAuth(user, password) if user else None
```

Comment:
requests supports just passing in the auth as a tuple, so this could be simplified to:
```
auth = (user, password) if user else None
```
and would mean the extra import is not needed.